qubes/app.py

```diff
@@ -427,6 +427,10 @@ class VMCollection:
     def __init__(self, app):
         self.app = app
         self._dict = {}
+        # Recently used disposable IDs: dispid -> destroy seconds since epoch
+        self._recent_dispIDs = {}
```

Comment:
Please don't use uppercase mixed with underscores, I don't see other parts of this file using it.

https://peps.python.org/pep-0008/#descriptive-naming-styles

`_recent_dispIDs` falls into the `Capitalized_Words_With_Underscored` category.

Reply:
> `_recent_dispIDs` falls into the `Capitalized_Words_With_Underscored` category.

Done.  Also renamed `self._no_dispID_reuse_period` to `self._no_dispid_reuse_period`